include/velocypack/Slice.h

```diff
@@ -58,6 +59,7 @@ class Slice {
   friend class ArrayIterator;
   friend class ObjectIterator;
 
+  // This variable should be accessed through the start() method
```

Comment:
I don't think this comment is justified. The `start()` method simply returns `_start` as is, and `_start` is a private member. So I don't see a great need for the VPackSlice class to use the `start()` method to access `_start`.

Reply:
It should, because otherwise ValueSlice would not work. ValueSlice overrides start() to point to the actual value, so that all inherited methods will also use it. Otherwise I would have to duplicate everything that uses _start in ValueSlice which is unreasonable.

Reply:
Ok, I overlooked this. Makes sense. If you could add that to the comment it would help me (and potentially others) to understand the implications of using `_start` vs. `start()`.